client/lib/posts/post-list-store.js

```diff
@@ -1,23 +1,28 @@
 /**
  * External dependencies
  */
-var debug = require( 'debug' )( 'calypso:posts-list' ),
-	clone = require( 'lodash/lang/clone' ),
-	assign = require( 'lodash/object/assign' ),
-	transform = require( 'lodash/object/transform' ),
-	difference = require( 'lodash/array/difference' ),
-	last = require( 'lodash/array/last' ),
-	max = require( 'lodash/collection/max' );
+import debugFactory from 'debug';
+const debug = debugFactory( 'calypso:posts-list' );
```

Comment:
I agree.  This matches precedent as well:  there are 44 instances of `import debug` or `import _debug` in the code, and only 6 are immediately followed by the `const debug` statement.

Because I was curious:

| String | Count |
| --- | --- |
| `import debugModule` | 25 |
| `import debugFactory` | 12 |
| `import _debug` | 4 |
| `import Debug` | 3 |